src/commands/install.js

```diff
@@ -63,7 +63,7 @@ export async function install(opts: InstallOptions) {
   });
 
   for (let workspace of workspaces) {
-    const dependencies = workspace.pkg.getAllDependencies().keys();
+    let dependencies = Array.from(workspace.pkg.getAllDependencies().keys());
```

Comment:
This was basically the whole thing

Reply:
Do you recall what the Array.from was for here @jamiebuilds ?